core/servers/chat/mrc_multiplexer.js

```diff
@@ -0,0 +1,325 @@
+/* jslint node: true */
+'use strict';
+
+//  ENiGMA½
+const Log                   = require('../../logger.js').log;
+const { ServerModule }      = require('../../server_module.js');
+const Config                = require('../../config.js').get;
+const { Errors }            = require('../../enig_error.js');
+const SysProps              = require('../../system_property.js');
+const StatLog               = require('../../stat_log.js');
+
+//  deps
+const net                   = require('net');
+const _                     = require('lodash');
+const os                    = require('os');
+
+
+// MRC
+const protocolVersion   = '1.2.9';
+const lineDelimiter     = new RegExp('\r\n|\r|\n');
+
+const ModuleInfo = exports.moduleInfo = {
+    name        : 'MRC',
+    desc        : 'An MRC Chat Multiplexer',
+    author      : 'RiPuk',
+    packageName : 'codes.l33t.enigma.mrc.server',
+    notes       : 'https://bbswiki.bottomlessabyss.net/index.php?title=MRC_Chat_platform',
+};
+
+const connectedSockets = new Set();
+
+exports.getModule = class MrcModule extends ServerModule {
+    constructor() {
+        super();
+
+        this.log            = Log.child( { server : 'MRC' } );
+        this.config         = Config();
+        this.mrcConnectOpts = {
+            host         : this.config.chatServers.mrc.serverHostname || 'mrc.bottomlessabyss.net',
+            port         : this.config.chatServers.mrc.serverPort || 5000,
+            retryDelay   : this.config.chatServers.mrc.retryDelay || 10000
+        };
+    }
+
+    _connectionHandler() {
+        const config = Config();
+        const boardName  = config.general.prettyBoardName || config.general.boardName;
```

Comment:
What's a pretty board name? (just curious)

Reply:
MRC needs two "formats" of your board name - one slugified which it uses as a "site name" in messages, and the "pretty" one, which can have pipe codes in and is shown when a user runs the `/BBSES` command. I didn't want to add it into `config.js` as it'd break pretty much everyone that didn't have it, but I thought a nice extra option to add for people that want to take advantage of it. What are your thoughts?

Reply:
Makes sense. I think you can add it to the config.hjson template + `config new` can populate it with the same value as `boardName` with a comment stating it can allow pipe color codes. It could be exposed and used elsewhere later (MCI code perhaps)